### DISCUSSION TITLE
Super clean code!

### DISCUSSION THREAD

**Discussion by Justus-M** [flyerhq/flutter_chat_ui#257] [Feedback]
It's incredible how readable and easy to modify this code is, especially in comparison to some other packages which even require licenses.

Very impressive quality overall, and high level of general professionalism from the team!

I'm still relatively new to flutter and app development (approx. 8 intense months) so I don't want to step on anyone's toes, but hopefully I can give back and contribute eventually.

**Comment by demchenkoalex** [comment]
Thank you very much! I think we can do even better, thinking about potential v2 :)
